client/components/app.jsx

```diff
@@ -24,22 +33,74 @@ class App extends React.Component {
       })
   }
 
+  nextPhoto() {
+    if(this.state.index === this.state.summary.photos.length - 1) {
+      return this.setState({
+        index: 0,
+        translateValue: 0
+      })
+    }
+    
+    this.setState(prevState => ({
+      index: prevState.index + 1,
+      translateValue: prevState.translateValue - this.photoWidth()
+    }));
+
+  }
+
+  photoWidth() {
+     return document.querySelector('.photo').clientWidth
+  }
+
+  prevPhoto() {
+    if(this.state.index !== 0) {
+      this.setState(prevState => ({
+        index: prevState.index - 1,
+        translateValue: prevState.translateValue + this.photoWidth()
+      }));
+    }
+  }
+
+  handleMouseEnter(e) {
+    e.target.className = 'hover';
+  }
+
   render() {
     return (
       <div>
         <h2 className="title">{this.state.summary.title.toUpperCase()} ({this.state.summary.releaseDate.slice(-4)})</h2>
+
         <div className="movie-box">
-          <div className="trailer"><img className="trailer-photo"src={this.state.summary.trailer}/></div>
+          <div><img className="trailer" onMouseEnter={(e) => this.handleMouseEnter(e)} src={this.state.summary.trailer}/></div>
           <div className="movie-details">
             <h2 className="release-date">{this.state.summary.releaseDate}</h2>
             <div className="rating-duration">{this.state.summary.rating}, {this.state.summary.duration}</div>
             <div className="genre">{this.state.summary.genre.toUpperCase()}</div>
             <h2 className="score"><span>🥛</span> {this.state.summary.score}%</h2>
           </div>
         </div>
-        <h3 className="synopsis-header">Synopsis</h3>
-        <div className="synopsis">{this.state.summary.synopsis}</div>
-        <div className="photos">{this.state.summary.photos}</div>
+
+        <div className="synopsis-box">
+          <h3 className="synopsis-header">Synopsis</h3>
+          <div className="synopsis">{this.state.summary.synopsis}</div>
+        </div>
+
+        <div className="carousel">
+          <LeftArrow prevPhoto={this.prevPhoto} />
+          <div className="photos-container">
+            <div className="photos"
+              style={{
```

Comment:
Styles handled in css preferable to inline styles. Might want to move this to stylesheet